src/twisted/plugins/cred_unix.py

```diff
@@ -52,7 +53,9 @@ class UNIXChecker(object):
 
     def checkPwd(self, pwd, username, password):
         try:
```

Comment:
This applies to all the methods in this class -- its a shame there are no method docstrings.  The fact that so much of Twisted is documented is one of (my opinion) its strengths.  Should lack of docsstrings live in another ticket?

Reply:
There are a few functions in this file which do not have docstrings.  I'm not going to add docstrings for all of them, but I agree with you that this could be improved, so I added docstring for `checkPwd`, `checkSpwd`, and `verifyCryptedPassword`.

Those are important functions in this file, so it doesn't hurt to document them a bit better.